orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationResponse.java

```diff
@@ -0,0 +1,26 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.orchestration.client.model.CompletionPostResponse;
+import javax.annotation.Nonnull;
+
+/** Orchestration chat completion output. */
+public class OrchestrationResponse extends CompletionPostResponse {
+  /**
+   * Get the message content from the output.
+   *
+   * <p>Note: If there are multiple choices only the first one is returned
+   *
+   * @return the message content or empty string.
+   * @throws OrchestrationClientException if the content filter filtered the output.
+   */
+  @Nonnull
+  public String getContent() throws OrchestrationClientException {
+    if (getOrchestrationResult().getChoices().isEmpty()) {
```

Comment:
Can this ever happen? How is it handled in our OpenAI implementation ?

Reply:
Removed (also in the Open AI implementation).